src/test/java/ParserTest.java

```diff
@@ -0,0 +1,31 @@
+import org.junit.jupiter.api.Test;
```

Comment:
Very comprehensive JUnit testing overall! Test methods were descriptive, hence it was easy to understand what tests you were trying to run.

Reply:
My suggestion for this commit is to follow standard convention for your commit message, eg. "Add JUnit testing"

Reply:
OKay! Will take note of my commit messages in the future HAHAHA thankyou!!